README.md

```diff
@@ -78,8 +81,10 @@ I'm looking for a full remote position (EU)! See [edoardottt.com/cv](https://www
 | Growblocks | All | Full remote | [Link](https://thehub.io/startups/growblocks) | ? |
 | Hacktive Security | All | Full remote | [Link](https://www.linkedin.com/company/hacktive-security/jobs/) | ? |
 | Heroku | Some | Full remote | [Link](https://www.heroku.com/careers/) | ? |
+| Horizon3.ai | Some | Full remote | [Link](https://www.horizon3.ai/join-our-team/) | ? |
 | Human Rights Watch | Some | Full remote | [Link](https://careers.hrw.org/) | ? |
 | Human Security | All | Full remote | [Link](https://www.humansecurity.com/company/careers) | ? |
+| Huntress | All | Full remote | [Link](https://www.huntress.com/company/careers) | ? |
```

Comment:
I think this should be `Full Remote US`

Reply:
I think this works, as they have UK and Australia remote positions listed at the bottom of their jobs page. Let me know if you feel otherwise, I can change. Thanks!

Reply:
it's okay. remeber to request a new review when changes are done :)